db/migrate/20170601062052_change_column_of_requests.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnOfRequests < ActiveRecord::Migration[5.0]
+  def change
+    change_column :requests, :status, 'integer USING CAST(status AS integer)'
```

Comment:
I thought same as you, but when I did migration with your line, I got error and googled error message, I reached this page. I know this line from this.

https://stackoverflow.com/questions/17075173/rails-migrations-change-column-with-type-conversion